fs/btrfs/zstd.c

```diff
@@ -37,10 +37,23 @@ struct workspace {
 	size_t size;
 	char *buf;
 	struct list_head list;
+	unsigned int level;
 	ZSTD_inBuffer in_buf;
 	ZSTD_outBuffer out_buf;
 };
 
+static int zstd_reallocate_mem(struct workspace *workspace, int size)
+{
+	void* new_mem = kvmalloc(size, GFP_KERNEL);
```

Comment:
Kernel style is `void *` (space before the asterisk, not after), and a newline after declarations. So

```
void *new_mem;

new_mem = kvmalloc(size, GFP_KERNEL);
```

Would be ideal.